lib/puppet/indirector/catalog/compiler.rb

```diff
@@ -288,8 +298,11 @@ def compile(node, options)
         end
 
         if checksum_type && config.is_a?(model)
-          str = _("Inlined resource metadata into static catalog for %{node}") % { node: node.name }
-          str += _(" in environment %{env}") % { env: node.environment } if node.environment
+          str = if node.environment
+                  _("Inlined resource metadata into static catalog for %{node} in environment %{environment}") % { node: node.name, environment: node.environment }
```

Comment:
Can we find a less jargon-y and more descriptive term than "inlined" ?

Reply:
Like, would "inserted" be accurate? I just don't see "inlined" being easily translated, and even for English speakers it is not great.

Reply:
We use inlined in the docs as well: https://puppet.com/docs/puppet/5.3/static_catalogs.html Probably best to keep it inlined for now.

Reply:
I think for the time being we were also trying to not really change the text of strings we're marking for internationalization, in case people are relying on the output in scripts (like we are in our tests). Though it might be good to keep track of changes we'd like to make, and do some clean up in the next major release.